Apps/Database/src/Delegates/DBEmailDelegate.cs

```diff
@@ -77,6 +77,45 @@ public List<Email> GetLowPriorityEmail(int maxRows)
             return retVal;
         }
 
+        /// <inheritdoc />
+        public List<Email> GetStandardPriorityEmail(int maxRows)
+        {
+            this.logger.LogTrace($"Getting list of standard priority emails from DB... {maxRows}");
+            List<Email> retVal = this.dbContext.Email.Where(p => p.EmailStatusCode == EmailStatus.New &&
+                                                   p.Priority >= EmailPriority.Standard && p.Priority < EmailPriority.High)
```

Comment:
as we are sending Standard ones why do we need "..p.Priority < EmailPriority.High"?
Should it just be "p.Priority == EmailPriority.Standard" ?

Reply:
I think the confusion here is that we use priorities in the database from 0 to 10 (if i remember correctly) but we only have 3 values in our EmailPriority Enum (Standard, High, Urgent) so there are numbers in between those three (lets say standard is 4 and high is 7, if i have an email priority that is 5 its considered as standard because its less than the high number. I don't think there's a problem but i'm sure we could have done this in a simpler way

Reply:
right admin client can only set 1 value for standard. The purpose of this email job is just to send that Standard email, nothing else right?

Reply:
It should cover all cases but I think that we could probably create a bug to update the DB to handle only the enumeration values.

Reply:
Hey - we do it this way so that a change in the numbers in the DB (i.e. if someone manually changed the priority to 150 it would still get sent) doesn't interfere with the functionality of the job

Reply:
It doesn't seem a good way to use enum here though. Enum is to limit the choices. Changing priority to 150 should not be expected unless we introduce a new priority here, i.e. "Super Urgent" = 150.
We can chat further on our call to make sure.